src/components/Admin/AddQuestion.js

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { compose } from "recompose";
```

Comment:
Isn't being used here, should be removed

Reply:
Resolved in commit 15bc7feade4e9e37cdcfb0f3f5de88ed6c4b5193